### PR TITLE
#fixes service code, mongo query change, redis connection,mongoose up…

### PR DESCRIPTION
**Pull request by Linoy339**
1. Fixes in Service class
2. Upgrade of mongoose
3. change in mongo query
4. redis connection string in bootstrap

### CODE REVIEW

File: src/service/CredentialService.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class CredentialService {
   public static async set(auth: any, type_id: string | null, access_key: string, credential: any | null) {
     const CredentialRepository = new Repository().getCredentialRepository()
     type_id = await _verify(auth, ["self", "parent"], type_id)
-    if (credential === null) {
+    if (credential !== null) {
```
Comment (avaidyam) [inline src/service/CredentialService.ts:33]: Can you keep the statement as `=== null` and swap the contents of the if and else?

File: src/service/ParticipantService.ts [hunk 2]
```diff
@@ -121,7 +121,7 @@ ParticipantService.Router.delete("/participant/:participant_id", async (req: Req
 })
 ParticipantService.Router.get("/participant/:participant_id", async (req: Request, res: Response) => {
   try {
-    let output = { data: ParticipantService.get(req.get("Authorization"), req.params.participant_id) }
+    let output = { data:await ParticipantService.get(req.get("Authorization"), req.params.participant_id) }
```
Comment (avaidyam) [inline src/service/ParticipantService.ts:124]: There should be a space after the colon for consistent formatting.

File: src/service/CredentialService.ts (hunk 1 above)
Comment (Linoy339) [reply src/service/CredentialService.ts:33]: ok

File: src/service/ParticipantService.ts (hunk 2 above)
Comment (Linoy339) [reply src/service/ParticipantService.ts:124]: Ok. We used to format it @avaidyam . This time, I have quickly done without any formatting. We will take care next time

Comment (avaidyam) [reply src/service/ParticipantService.ts:124]: No problem for now. Perhaps we should integrate eslint and prettier into the build/run phase like was done for the frontend code. (Linting issues will cause warnings/errors and fail to build the image.)

Comment (Linoy339) [reply src/service/ParticipantService.ts:124]: Ok. We can do it

**Pull request merged**